plugin-server/src/index.ts

```diff
@@ -34,6 +35,9 @@ const status = new Status(alternativeMode)
 
 status.info('⚡', `@posthog/plugin-server v${version}`)
 
+status.info('🩺', 'Starting health check endpoint')
+startServer(status)
+
```

Comment:
Not sure if this HTTP server should be started for every `alternativeMode` below...
I'd move it deeper into the actual plugin server code, somewhere in the `hub` perhaps?